README.md

```diff
@@ -123,6 +123,7 @@ Now create an index.html file with the following contents:
     <script>
       var model = new falcor.Model({source: new falcor.HttpDataSource('/model.json') });
 
+      model.
       // retrieve the "greeting" key from the root of the Virtual JSON resource model.
```

Comment:
Nitpick: I'm fairly sure the "model." at the end of line 127 is what was intended to be moved to the next line, rather than needing a preceding one on 126. Mind doing that? Otherwise thanks a lot for the PR!